rcl/test/rcl/test_graph.cpp

```diff
@@ -1065,7 +1070,7 @@ class NodeGraphMultiNodeFixture : public TestGraphFixture
       ASSERT_EQ(RCUTILS_RET_OK, rcutils_string_array_fini(&node_names));
       ASSERT_EQ(RCUTILS_RET_OK, rcutils_string_array_fini(&node_namespaces));
       ASSERT_LE(attempts, max_attempts) << "Unable to attain all required nodes";
-    } while (last_size < 3u);
+    } while (last_size < 2u);
```

Comment:
I think the test should make sure that the two nodes found are named `remote_node` and `node`, otherwise this check might succeed if `node` and `old_node` are present.

Reply:
Done! https://github.com/ros2/rcl/pull/1192/commits/996b58800eab21b3e6493ff5b845ce22c6729bd7